testController.js

```diff
@@ -1,56 +1,68 @@
+    var cityName = document.getElementById('city-name');
+    var weatherDescription = document.getElementById('weather-description');
+    var currentTemp = document.getElementById('weather-temps');
+    var cityInputButton = document.getElementById('enter-city');
+    var seeForecastButton = document.getElementById('see-forecast');
+    var forecastInformation = document.getElementById('forecast');
 
     var getTodayWeather = function () {
         document.getElementById('forecast').style.display = 'none'
-        getWeather(`${API_URL}/data/2.5/weather?q=${getCity()}&appid=${OPENWEATHER_KEY}`).then(function (data) {
+        getWeather(`${API_URL}${TODAY_WEATHER_ROUTE}${getCity()}${OPENWEATHER_KEY}`).then(function (data) {
             let weatherToday = data;
-            document.getElementById('info-wrapper').style.display = 'block';
-            document.getElementById('city-name').textContent = `Град: ${weatherToday.name}`;
-            document.getElementById('weather-description').textContent = `Днес времето ще бъде ${weatherToday.weather[0].description}`;
-            document.getElementById('weather-temps').textContent = `Температурата в момента е ${Math.round(weatherToday.main.temp - KELVIN_CONSTANT)} градуса`
+            infoWrapper.style.display = 'block';
+            cityName.textContent = `Град: ${weatherToday.name}`;
+            weatherDescription.textContent = `Днес времето ще бъде ${weatherToday.weather[0].description}`;
+            currentTemp.textContent = `Температурата в момента е ${Math.round(weatherToday.main.temp - KELVIN_CONSTANT)} градуса`
         }).catch(function (error) {
             throw new Error(error.message)
         })
 
     }
-    document.getElementById('enter-city').addEventListener('click', getTodayWeather)
+    cityInputButton.addEventListener('click', getTodayWeather)
 
 
     var getThreeDayForecasat = function () {
-        getForecast(`${API_URL}/data/2.5/forecast?q=${getCity()}&appid=${OPENWEATHER_KEY}`).then(function (data) {
-            if (data.list[0]) {
-                document.getElementById('forecast').style.display = 'block';
-                document.getElementById('img-day-one').src = `${API_URL}/img/w/${data.list[0].weather[0].icon}.png`
-                document.getElementById('day-one').textContent = `Утре времето ще бъде ${data.list[0].weather[0].description}`;
-                document.getElementById('day-one-temp').textContent = `Със средна температура от ${Math.round(data.list[0].main.temp - KELVIN_CONSTANT)} градуса`;
-            } else {
-                document.getElementById('day-one').textContent = `Липсва информация за този ден`
-            }
+            forecastInformation.innerHTML = '';
+        getForecast(`${API_URL}${FORECAST_ROUTE}${getCity()}${OPENWEATHER_KEY}`).then(function (data) {
+            forecastInformation.style.display = 'block';
+            for (let day = 0; day < howManyDays(); day++) {
```

Comment:
Appending in a loop is not the most efficient way to do it. Because on every iteration you are accessing the DOM. An alternative approach can be as follows:
- Create wrapper, append all elements in the loop and the in one step append this wrapper or set it as inner html of the target parent element.